src/main/java/com/trustedchoice/askkodiak/v2/client/AskKodiak.java

```diff
@@ -140,7 +219,7 @@
      * @throws AskKodiakException error
      */
     @RequestLine("GET /v2/products/class-code/naics/{code}")
-    Products getProductsEligibleForCode(@Param("code") String code, @QueryMap EligibleQuery query) throws AskKodiakException;
+    Products getProductsEligibleForCode(@Param("code") String code, @QueryMap(encoded = true) Map<String, Object> query) throws AskKodiakException;
```

Comment:
I have some concern about creating two `getProductsEligibleForCode()` methods with such similar signatures and different functionality. I think it will cause confusion for users of the SDK. Maybe change the name of this method so we know that `Map` `query` needs to be encoded.